backend/src/main/java/gov/cdc/usds/simplereport/db/repository/AdvisoryLockManager.java

```diff
@@ -0,0 +1,39 @@
+package gov.cdc.usds.simplereport.db.repository;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.jpa.repository.query.Procedure;
+
+/**
+ * A mix-in repository for entity repositories that manage processes that use
+ * advisory locks.
+ */
+public interface AdvisoryLockManager {
+
+    int CORE_API_LOCK_SCOPE = 110506458; // some arbitrary 32-bit number that defines "our" locks
```

Comment:
My IDE complains about constants defined in an interface like this.

Reply:
So does Sonar, usually. I think this is a reasonable exception to the rule they are advocating for, since there are no implementing classes to put the constants in. But I do see their point.